pages/api/models/selfCheckGroup.ts

```diff
@@ -0,0 +1,26 @@
+import { model, Schema } from "mongoose";
+
+interface SelfCheckGroupInterface {
+    _id: Schema.Types.ObjectId;
+    questionIDs: [Schema.Types.ObjectId];
+}
+
+const SelfCheckGroupSchema = new Schema<SelfCheckGroupInterface>(
+    {
+        questionIDs: {
+            type: [{ type: Schema.Types.ObjectId, ref: "Questions" }],
+            default: [],
+        },
+    },
+    {
+        timestamps: {
+            currentTime: Date.now,
+            updatedAt: "dateUpdated",
+            createdAt: "dateCreated",
+        },
+    },
```

Comment:
`{ timestamps: true }` should be sufficient (https://masteringjs.io/tutorials/mongoose/timestamps)